jenkins/job-configs/kubernetes-jenkins-pull/credentials.yaml

```diff
@@ -0,0 +1,7 @@
+- wrapper:
+    name: e2e-credentials-binding  # TODO(fejta): make this match non-pull
+    wrappers:
+        - credentials-binding:
+            - file:
+                credential-id: 'kubekins@kubernetes-jenkins.iam.gserviceaccount.com'
```

Comment:
you may just want to use the env-inject plugin, like https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml#L200-L204